test/test_bito.py

```diff
@@ -180,6 +182,17 @@ def test_gp_instance():
 
     edge_idx_to_pcsp_map = inst.build_edge_idx_to_pcsp_map()
     print("edge_pcsp_map:", edge_idx_to_pcsp_map)
+
+    nni_engine = inst.get_nni_engine()
+    nni_engine.run_init()
+    adj_nni_count = nni_engine.get_adjacent_nni_count()
+    nni_engine.run()
+
+    dag = inst.get_dag()
+    node_count = dag.get_node_count()
+    edge_count = dag.get_edge_count()
+    print("nni_stats: ", adj_nni_count, node_count, edge_count)
+
```

Comment:
Is this mostly a demo vs a test? Do we want to have test conditions? Also, isn't the pass just below superfluous?

Reply:
Yes, this is just a demo.